h/api/models/annotation.py

```diff
@@ -86,6 +86,17 @@ class Annotation(Base, mixins.Timestamps):
                                    viewonly=True,
                                    uselist=False)
 
+    @property
+    def target_links(self):
+        """A list of the URLs to this annotation's targets."""
+        links = []
+        for target_selector in self.target_selectors:
+            source = target_selector.get("source")
```

Comment:
Ah, it looks like the `target` object submitted by the client is a list of dicts with `selector` and `source` fields. These used to all be stored in legacy Es, and `h.api.models.elastic.Annotation.target_links` returns a list of all these `source` fields. But _now_ we discard all targets other than the first, and we discard all fields of the first target other than `selector`. So `source` is discarded and not stored in Postgres, even though the client still sends it.

I guess we should just delete this `target_links` property and remove the code from feeds that uses it